features/ouders.feature

```diff
@@ -2,113 +2,349 @@
+    | 52        | Me'Lisa            | 20170106                                           |                 |
+    | 52        |                    | 00000000                                           | O               |
+    | 3         | Kevin              | 20190317                                           |                 |
+    | 53        |                    |                                                    |                 |
+    | 53        |                    | 00000000                                           | O               |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                                                                                      |
+    | type                | RaadpleegMetBurgerservicenummer                                                             |
+    | burgerservicenummer | 999996629                                                                                   |
+    | fields              | ouders.ouderAanduiding,ouders.naam.voornamen,ouders.datumIngangFamilierechtelijkeBetrekking |
+    Dan heeft de ouder met ouderAanduiding 'ouder1' de volgende gegevens
+    | naam                                          | waarde     |
+    | type                                          | Ouder      |
+    | ouderAanduiding                               | ouder1     |
+    | datumIngangFamilierechtelijkeBetrekking.datum | 2019-03-17 |
+    En heeft de ouder met ouderAanduiding 'ouder1' de volgende 'naam' gegevens
+    | naam      | waarde           |
+    | voornamen | Olivia Ghislaine |
+    En heeft de ouder met ouderAanduiding 'ouder2' de volgende gegevens
+    | naam                                          | waarde     |
+    | type                                          | Ouder      |
+    | ouderAanduiding                               | ouder2     |
+    | datumIngangFamilierechtelijkeBetrekking.datum | 2019-03-17 |
+    En heeft de ouder met ouderAanduiding 'ouder2' de volgende 'naam' gegevens
+    | naam      | waarde    |
+    | voornamen | Kevin     |
+
+  @gba
+  Scenario: ontkenning gevolgd door erkenning
+    Gegeven de persoon met burgerservienummer 999996162 heeft de volgende ouders in de registratie
+    | Categorie | Voornamen (02.10)  | Onjuist (84.10) |
+    | 2         | Chantal            |                 |
+    | 3         | Wieger             |                 |
+    | 53        |                    |                 |
+    | 53        | Mark               | O               |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                                              |
+    | type                | RaadpleegMetBurgerservicenummer                     |
+    | burgerservicenummer | 999996162                                           |
+    | fields              | ouders.ouderAanduiding,ouders.naam.voornamen,ouders |
```

Comment:
Is de waarde ',ouders' aan het eind van de fields parameter wel correct?
Die zou er immers voor zorgen dat alle properties van 'ouders' dus ook 'ouderAanduiding' en 'naam.voornamen' al opgenomen worden.

Reply:
klopt, is dubbelop. verwijderd